### ISSUE TITLE
[Table] action button accessibility violations

### ISSUE THREAD

**Issue by tay1orjones** [IBM/carbon-addons-iot-react#1303]
### Describe the bug

Originally reported by @JordanWSmith15 in [internal Watson-IoT/pal-tracking#439](https://github.ibm.com/Watson-IoT/pal-tracking/issues/439):

> There are a few accessibility violations issues with the table action buttons
> 1.  Go to https://carbon-addons-iot-react.com/?path=/story/watson-iot-table--stateful-example-with-expansion-maxpages-and-column-resize
> 2. Open Accessibility tab in storybook
> Result: There are three violations involving the button row
